docs/tutorials/tutorial_observables.ipynb

```diff
@@ -0,0 +1,1572 @@
+{
```

Comment:
<p>I think using weather as an observable might be a bit confusing since I would expect this to change over time. Maybe we could use climate instead?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenSourceEconomics/respy/pull/367/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>